src/tooltip.ts

```diff
@@ -362,8 +362,14 @@ const tooltipPlugin = ViewPlugin.fromClass(class {
   }
 
   maybeMeasure() {
+    if (this.manager.tooltipViews.length) {
+      for (let t of this.manager.tooltipViews) {
+        t.dom.classList.toggle('mod-hidden', !this.view.hasFocus);
+      }
```

Comment:
Move this to inside the next block? From a reflow perspective I would suggest putting all the DOM manipulation code as close as possible, so I suggest moving it to after the requestMeasure call.